### DISCUSSION TITLE
请问如何将v2ray-core 作为一个sdk 集成到我自己到app 中，哪位大神能指点一个思路？谢谢

### DISCUSSION THREAD

**Discussion by fuqiangleon** [v2fly/v2ray-core#835] [Ideas]
请问如何将v2ray-core 作为一个sdk 集成到我自己到app 中，哪位大神能指点一个思路？谢谢

**Comment by Cwek** [comment]
我记得原项目维护写过一篇文章是将v2ray核心作为lib组件引用的

参考这个项目： https://github.com/v2ray/v2ray-plugin
是ss-android实现v2ray底层传输插件

**Comment by rurirei** [comment]
there are much projects you should find, just make search.
